src/components/flowchart/draw.js

```diff
@@ -301,12 +320,17 @@ export const drawEdges = function (changed) {
   const exitEdges = this.el.edges.exit();
   const allEdges = this.el.edges.merge(enterEdges).merge(exitEdges);
 
-  if (changed('edges')) {
-    enterEdges
-      .append('path')
+  if (changed('edges', 'newParamsFlag', 'focusMode', 'inputOutputDataNodes')) {
```

Comment:
qq: do you think we should remove `mewParamsFlag` from this PR since its removal has already been merged? Or should that be a follow up PR?

Reply:
I can remove this in this PR :) thanks for the reminder!